internal/key/key.go

```diff
@@ -15,15 +15,15 @@ import (
 const (
 	// IDChars represents the character set used to generate cluster IDs.
 	// (does not contain 1 and l, to avoid confusion)
-	idChars = "023456789abcdefghijkmnopqrstuvwxyz"
+	IDChars = "023456789abcdefghijkmnopqrstuvwxyz"
```

Comment:
IMO we can include `1` if `l` is not included.

Reply:
This can be discussed later. Let's keep it here as it is currently IMO